cursorless-talon/src/modifiers/containing_scope.py

```diff
@@ -45,18 +45,30 @@
     "end tag": "xmlEndTag",
 }
 
+select_multiple_modifiers = {
+    "every", 
+    "all"
+}
+
 
-@mod.capture(rule="[every] {user.cursorless_scope_type}")
+@mod.capture(rule="[(every|all)] {user.cursorless_scope_type}")
```

Comment:
Would be nifty if we could put this into a capture against the set above.

Reply:
You could use a Talon list, no?